projects/env-semconv.md

```diff
@@ -0,0 +1,58 @@
+### Description
+
+This working group looks to identify and standardize observability for
+sustainability purposes with regards to infrastructure and applications. This
+would look at current tooling around sustainability metrics and standardize the
+meaning and approach for these metrics.
+
+Current efforts of implementations:
+
+- [Kepler](https://github.com/sustainable-computing-io/kepler)
+- [Scaphandre](https://github.com/hubblo-org/scaphandre)
+
+We want to have a unified metrics convention in order to better facilitate measuring sustainability metrics across different ecosystems
+
+### Deliverables
+
+- Semantic conventions for sustainability metrics
+- Reference Prototypes metrics
+- Recommended practices for reporting on sustainability metrics 
```

Comment:
I see "recommended practices" is mentioned here, but not in the timeline. Can this be scoped a little tighter? Is this related to "how" to measure or "what/where" to measure for instance? What form will these recommendations be given as?

Reply:
I think this is part of the discussion to be had. I would go to "the recommended way to measure a metric", but think thats up for discussion

Reply:
Thanks for putting this on the time line.

Given that there are many misconceptions and wrong assumptions around "sustainability", those recommended practices could also offer some guidelines for correctly interpreting those metrics.

Reply:
That makes sense. I just wanted to discuss the potential scope creep in terms of the guidance that OpenTelemetry can give. For instance, what sustainability metrics each organisation should pay more attention to, depending on their workloads and data centre placement, or how to interpret them, is something that other groups are probably better place at answering. However, as that will be discussed as part of this group I'm happy with the answer "within the scope of OpenTelemetry".